src/client/Microsoft.Identity.Client/Account.cs

```diff
@@ -31,6 +31,16 @@ public Account(string homeAccountId, string username, string environment)
 
         public AccountId HomeAccountId { get; }
 
+        public string UniqueId { get; internal set; }
+
+        public string DisplayableId { get; internal set; }
```

Comment:
I don't see these optional fields being used anywhere, why are they needed?

Reply:
They are not. this is dead code